src/providers/mbed_provider/key_management.rs

```diff
@@ -13,14 +11,16 @@ use parsec_interface::operations::{
     psa_destroy_key, psa_export_public_key, psa_generate_key, psa_import_key,
 };
 use parsec_interface::requests::{ProviderID, ResponseStatus, Result};
+use psa_crypto::operations::key_management as new_key_management;
```

Comment:
I wasn't sure what to do about this rename. I thought leaving it as `use psa_crypto::operations::key_management;` might get confusing as it is already in a file called `key_management.rs`. Thoughts? Maybe `psa_crypto_key_management`?